src/client/index.ts

```diff
@@ -14,7 +14,6 @@ import {
 } from "../tokenLookup";
 import OnChainTokenModule from "./../onChainTokenModule";
 import Web3WalletProvider from "./../utils/Web3WalletProvider";
-import "./../theme/style.css";
```

Comment:
Thanks for providing more detail around the issues from the Next JS forum about this (https://github.com/vercel/next.js/issues/19936) - the reasoning makes sense, even though it incurs more work for web developers.

To implement this change can you update the ReadMe with steps for the developers to install the CSS alongside the core lib.

Reply:
@nicktaras yes, readme now updated.